corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,27 +34,32 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
     <div data-bind="saveButton: saveButton"></div>
     <div class="row">
         <div class="col-md-10">
-            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">Export to Excel</a>
-            <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">Import from Excel</a>
+            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">{% trans "Export to Excel" %}</a>
+            <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">{% trans "Import from Excel" %}</a>
             <h3 data-bind="text: $root.activeCaseType()"></h3>
+            <br/>
+            <button data-bind="click: $root.showDeprecated, visible: !showAll()" class="btn btn-info">{% trans "Show Deprecated Properties" %}</button>
```

Comment:
And all deprecate does is show/hide from this page, right? Could just say "hide"...